wicket/src/events.rs

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+use crate::state::ComponentId;
+use tokio::time::Instant;
+use wicketd_client::types::RackV1Inventory;
+
+use crossterm::event::Event as TermEvent;
+
+/// An event that will update state in the wizard
```

Comment:
are we still calling it a wizard?

Reply:
Fixed in [fd4ea2d](https://github.com/oxidecomputer/omicron/pull/2440/commits/fd4ea2d1c6e5f6a298b987e48eeefb5ee6bbeaf0)